easybuild/easyconfigs/m/MinPath/MinPath-1.6-intel-2021b.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'Tarball'
+
+name = 'MinPath'
+version = '1.6'
+
+homepage = 'https://omics.informatics.indiana.edu/MinPath'
+description = """MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions
+ using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological
+ pathways for a query dataset."""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
+
+sources = [{
+    'filename': 'minpath%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/mgtools/',
+        'repo_name': 'MinPath',
+        'commit': '46d3e81a4dca2310d558bea970bc002b15d44767',
```

Comment:
Keep track of the commit above, right below the `version` line, and add a comment that explain on how the commit was picked:

```python
version = '1.6'
# no tagged commit for MinPath 1.6, commit was determined based on commit history
# along with comments on top of MinPath.py
local_commit = '46d3e81a4dca2310d558bea970bc002b15d44767'
```

We should avoid using `git_config` when possible, since we can't reliable include a SHA256 checksum then (see discussion in https://github.com/easybuilders/easybuild-framework/issues/2726).

You can just download a `.tar.gz` straight from GitHub for a specific commit though:

```python
source_urls = ['https://github.com/mgtools/MinPath/archive/']
sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
```

Note that the checksum for the source file will change, so replace it.

Reply:
To be more exact, like this:

```diff
diff --git a/easybuild/easyconfigs/m/MinPath/MinPath-1.6-intel-2021b.eb b/easybuild/easyconfigs/m/MinPath/MinPath-1.6-intel-2021b.eb
index f3ab134953..5918acf2dd 100644
--- a/easybuild/easyconfigs/m/MinPath/MinPath-1.6-intel-2021b.eb
+++ b/easybuild/easyconfigs/m/MinPath/MinPath-1.6-intel-2021b.eb
@@ -2,6 +2,7 @@ easyblock = 'Tarball'

 name = 'MinPath'
 version = '1.6'
+local_commit = '46d3e81a4dca2310d558bea970bc002b15d44767'

 homepage = 'https://omics.informatics.indiana.edu/MinPath'
 description = """MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions
@@ -10,17 +11,11 @@ description = """MinPath (Minimal set of Pathways) is a parsimony approach for b

 toolchain = {'name': 'intel', 'version': '2021b'}

-sources = [{
-    'filename': 'minpath%(version)s.tar.gz',
-    'git_config': {
-        'url': 'https://github.com/mgtools/',
-        'repo_name': 'MinPath',
-        'commit': '46d3e81a4dca2310d558bea970bc002b15d44767',
-    },
-}]
+source_urls = ['https://github.com/mgtools/MinPath/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
 patches = ['MinPath-%(version)s_fix-glpsol-path.patch']
 checksums = [
-    '8ae836a0223f49d81991a16f9b5ae0d9a2d2f129dd345355fc04047d655cf8db',  # minpath1.6.tar.gz
+    '57e6b55cd00507578bad7528c60e3ad055311f62db928bd7c8de793513cec5a3',  # MinPath-1.6.tar.gz
     '4bf2c9ca9cb67ab4d5751b0af85df73c5cd7e368fc54737bab0907ce5a06a640',  # MinPath-1.6_fix-glpsol-path.patch
 ]
```